.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+    "rules": {
+        "semi": ["error", "always"],
+        "quotes": ["error", "double"],
+        "no-unused-vars": ["error", { "vars": "all", "args": "all" }],
+        "curly": ["error", "all"]
+    },
+    "extends": ["eslint:recommended", "react-app"]
+}
```

Comment:
This list can absolutely be longer, but it is more than sufficient for this project at this stage in development.
The point is not to mail down every single rule. Just start with some sane defaults and you'll update the rules later on (if needed).

Reply:
+1

Reply:
should we be throwing a `warning` instead of `error` on curly?